src/protocols/rdp/client.c

```diff
@@ -155,6 +157,12 @@ int guac_client_init(guac_client* client, int argc, char** argv) {
     pthread_mutexattr_settype(&(rdp_client->attributes),
             PTHREAD_MUTEX_RECURSIVE);
 
+    /* Initalize the lock */
+    pthread_rwlockattr_init(&lock_attributes);
+    pthread_rwlockattr_setpshared(&lock_attributes, PTHREAD_PROCESS_SHARED);
```

Comment:
Does this lock need `PTHREAD_PROCESS_SHARED`? I don't believe instances of this lock will be shared across different processes, just among threads of the same process.

Reply:
You are right. Removed lock attributes altogether.